### ISSUE TITLE
Feature request: Ability to mark stuff with custom markers

### ISSUE THREAD

**Issue by JMoVS** [toolbox4minecraft/amidst#4]
It'd be very helpful to. sort of like player's are displayed, set custom marker points in the map and name them so one could keep track of structures oneself built.

**Comment by stefandollase** [comment]
Thanks, this sounds indeed very helpful. I can imagine using this for my own world. I guess, the feature would only be available if the map is loaded from a world directory. Also, the custom world icons would be store in the world directory, in a separate sub directory.

However, now is not the right time to add more features. We currently try to find and fix remaining bugs and to merge Amidst with AmidstExporter, so we can create a new stable release that contains all features of both tools. Afterwards, we will be able to add new features.

**Comment by JMoVS** [comment]
Yeah, no problem. I just followed the readme ;-) No hurries, merge the other thing in first and then we can talk about feature requests.

**Comment by calvinjuarez** [comment]
Perhaps AMIDST could also find signs that have text on them, or that start with "!" or "@" or something.  Just a thought.

**Comment by stefandollase** [comment]
@JMoVS Don't get me wrong, I appreciate the suggestion. The only thing I was saying is, that it won't be implemented for quite some time. However, it will be implemented eventually.

@calvinjuarez I also like the idea of displaying signs (and maybe editing) signs. However, I would like it even more, if you would create a new issue for this suggestions, so we can keep things organized ;-)

**Comment by calvinjuarez** [comment]
@stefandollase Will do.
